doozer/doozerlib/cli/release_gen_assembly.py

```diff
@@ -42,8 +42,7 @@ def releases_gen_assembly(ctx, name):
 @click.option("--custom", default=False, is_flag=True,
               help="If specified, weaker conformance criteria are applied "
                    "(e.g. a nightly is not required for every arch).")
-@click.option("--prerelease", is_flag=True,
-              help="Prepare the assembly for a prerelease operator advisory. Only valid for preview and candidate assemblies.")
+@click.option('--pre-ga-mode', help='Prepare the advisory for "prerelease" or "advance" operator release')
```

Comment:
It would be nice to use `click.choices(["advance", "prerelease"])` here, as we do in other places

Reply:
added in https://github.com/openshift-eng/art-tools/pull/339/commits/17382dc779f01877da2a6a4424892e517a38b0c9